Makefile

```diff
@@ -92,11 +92,13 @@ generate-deepcopy: $(CONTROLLER_GEN)
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./pkg/apis/..."
 
 .PHONY: generate
-generate: generate-crds generate-deepcopy generate-kustomize generate-prometheus-rules docs
+generate: generate-crds generate-deepcopy generate-kustomize generate-prometheus-rules bundle docs
```

Comment:
doesn't this create a circular dependency? `generate -> bundle -> generate` ?

Reply:
Yup you are correct of course. Tbh I'm not exactly sure why I added this :thinking: 

Removed that dependency.